### ISSUE TITLE
False-positive warning around shorthand-values

### ISSUE THREAD

**Issue by zslabs** [sasstools/sass-lint#669]
Hi,
I have the following block:

`margin: calc(#{$doc-header-height} + #{$global-whitespace--regular}) 0 $global-whitespace--regular 0;`

This throwing the following error:

```
warning  Property `margin` should be written more concisely as ` 0 $global-whitespace--regular` instead of ` 0 $global-whitespace--regular 0`  shorthand-values
```

I have verified that the end-result of the `calc()` is: `margin: 76px 0 16px;`. Are you able to duplicate the same/ Thanks!

**Comment by DanPurdy** [comment]
Hi @zslabs 

Thanks for reporting this, I've managed to recreate and also produced a fix in #673 It seemed we were a) not correctly looking at functions being used as values in condensable properties and also due to the AST changes recently we weren't looking at interpolated values either (as the AST failed on them previously!).

**Comment by zslabs** [comment]
Awesome, thanks!
